src/include/rbd_types.h

```diff
@@ -77,6 +77,7 @@
 #define RBD_GROUP_HEADER_PREFIX "rbd_group_header."
 
 #define RBD_GROUP_DIRECTORY "rbd_group_directory"
+#define RBD_GROUP_REF "rbd_group_ref"
```

Comment:
Minor: keep this implementation detail hidden in cls/rbd. Also, can we rename it to RBD_IMAGE_GROUP_SPEC or something similar to denote that it's tied to the image and it links to a group spec?